data/carousel.toml

```diff
@@ -0,0 +1,72 @@
+[[items]]
+title = "Spotlight PA Featured Impact"
+tagline = "Housing"
+text = "A Spotlight PA investigation into Pennsylvania’s pandemic mortgage relief program found that long delays and poor communication were leaving thousands in limbo, with their homes and utilities on the line. The story prompted the state to fire the third-party contractor that was running the program."
+short = "The state fired a third-party contractor after Spotlight PA revealed serious issues with Pennsylvania’s pandemic mortgage relief program."
+link = "https://www.spotlightpa.org/series/the-waiting-game/"
+linkText = "Read the series"
+image = "external/s36w4vxgrg2nnpxjrf10fce26m.jpeg"
+width = "2400"
```

Comment:
I think it probably makes more sense to leave the sizes out of the data and just have all of the carousel images resized to something reasonable like 1200x800.